Makefile

```diff
@@ -13,8 +13,9 @@ help:
 bindings: ## generates contract bindings
 	cd contracts && rm -rf bindings/* && ./generate-bindings.sh 
 
-mocks: ## generates mocks
+generate: ## generates mocks and wire files
```

Comment:
why do we need to couple these 2 generation? any particular reason or just to combine generation together?

Reply:
Is there a way to separate go generated stuff? Right now if we run go generate it just looks for all the go:generate comments in the repo and executed all of them. Maybe there’s a way to tag the generate commands with subtask names and get go generate to filter on those?